content/zh/docs/concepts/architecture/nodes.md

```diff
@@ -26,6 +26,10 @@ architecture design doc for more details.
 -->
 在 Kubernetes 中，节点（Node）是执行工作的机器，以前叫做 `minion`。根据你的集群环境，节点可以是一个虚拟机或者物理机器。每个节点都包含用于运行 [pods](/docs/concepts/workloads/pods/pod/) 的必要服务，并由主控组件管理。节点上的服务包括 [容器运行时](/docs/concepts/overview/components/#node-components)、kubelet 和 kube-proxy。查阅架构设计文档中 [Kubernetes 节点](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md#the-kubernetes-node) 一节获取更多细节。
 
+[scheduled](/docs/concepts/scheduling/)
```

Comment:
You have to use '/zh/docs/concepts/scheduling/' to navigate to the localized version, or you need to add logics to the render-link.html code below to get this done automagically.

Reply:
The snippet of the generated HTML page source:
```
<h1>节点</h1>



<!--
A node is a worker machine in Kubernetes, previously known as a `minion`. A node
may be a VM or physical machine, depending on the cluster. Each node contains
the services necessary to run [pods](/docs/concepts/workloads/pods/pod/) and is managed by the master
components. The services on a node include the [container runtime](/docs/concepts/overview/components/#node-components), kubelet and kube-proxy. See
[The Kubernetes Node](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md#the-kubernetes-node) section in the
architecture design doc for more details.
-->
<p>在 Kubernetes 中，节点（Node）是执行工作的机器，以前叫做 <code>minion</code>。根据你的集群环境，节点可以是一个虚拟机或者物理机器。每个节点都包含用于运行 

<a href="/zh/docs/concepts/workloads/pods/pod/">pods</a>
 的必要服务，并由主控组件管理。节点上的服务包括 

<a href="/zh/docs/concepts/overview/components/#node-components">容器运行时</a>
、kubelet 和 kube-proxy。查阅架构设计文档中 

<a href="https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md#the-kubernetes-node" target="_blank">Kubernetes 节点</a>
 一节获取更多细节。</p>
<p>

<a href="/zh/docs/concepts/scheduling/">scheduled</a>



<a href="/zh/docs/concepts/#kubernetes-control-plane">Kubernetes control plane</a>



<a href="/zh/docs/admin/kubelet-authentication-authorization/">Kubelet authentication and/or authorization</a>
</p>
```

Reply:
The preview I saw is still something like this:
```
<p><a href="/docs/concepts/scheduling/">scheduled</a>
--
  | <a href="/docs/concepts/#kubernetes-control-plane">Kubernetes control plane</a>
  | <a href="/docs/admin/kubelet-authentication-authorization/">Kubelet authentication and/or authorization</a></p>
```

Maybe I was looking at the wrong place?

Reply:
Sorry for the confusion. I am testing locally with the latest version of Hugo.
The site is not set up to test this version.

Reply:
Just updated netlify.toml to use latest version of Hugo.
You should still see some links with problems.

https://deploy-preview-19830--kubernetes-io-master-staging.netlify.com/zh/docs/concepts/architecture/nodes/

Reply:
Now I can see the '/zh/' prefix is automatically added. The limits I saw so far: 1. intra-page links; 2. no auto fallback to English version.

Reply:
Yes. I guess the intra-page links can be corrected in the template. Wish there was a more "standard" way (integrated into Hugo) of creating links for multi-language doc sites.